stigaview_static/main.py

```
@@ -1,19 +1,34 @@
 
-def parse_args() -> argparse.Namespace:
+
+def _log_level_type(value: str) -> int:
+    try:
+        level = logging.getLevelName(value.upper())
+        # getLevelName returns the string if not found, so check if it's an integer
+        if isinstance(level, int):
+            return level
+        raise argparse.ArgumentTypeError(f"Invalid log level: {value}")
+    except AttributeError:
+        raise argparse.ArgumentTypeError(f"Invalid log level: {value}")
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)